src/contracts/core/DelegationManager.sol

```diff
@@ -685,6 +699,26 @@ contract DelegationManager is
         _beaconChainSlashingFactor[staker] = bsf;
     }
 
+    /// @dev Calculate amount of withdrawable shares from an operator for a strategy that is still in the queue
+    /// and therefore slashable.
+    function _getSlashableSharesInQueue(address operator, IStrategy strategy) internal view returns (uint256) {
```

Comment:
can we expose a public view func for this?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/881/commits/138eef799d1ce06c60ca1f2b06b626ff2410dfa5